termux/colors.properties

```diff
@@ -0,0 +1,41 @@
+# Eye Friendly Colors
+# Created by https://github.com/decaycs
+
+# Special
+foreground #a9b1d6
```

Comment:
i think this can look better if you use the original foreground color: `#b6beca`